.github/pull_request_template.md

```diff
@@ -32,3 +32,4 @@
 - [ ] I have not added any markdown to labels, titles and button text in config
 - [ ] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
 - [ ] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`
+- [ ] I have updated any relevant `.env` values in both `.env-template` and in Bitwarden
```

Comment:
[dust] I have two additional priorities here:

1. Increasing the likelihood that folks fill out this form.
2. Making sure this section is still readable.

What do you think about separating the checkboxes into sections, whether required/optional or api/web or some other way to break out similar checkboxes to make this section more readable? Do you think it would increase the likelihood that engineers fill out this form?

Reply:
> Increasing the likelihood that folks fill out this form
🤝 🙌 👍 👐